docs/examples/promise/README.md

```diff
@@ -1,19 +1,25 @@
 # ApiPromise Examples
 
-Here you will find a list of examples that takes you through the basics of connecting to a local node, retrieving data from the Node and chain and execute transactions on the chain. It uses the [[ApiPromise]] interface.
+Here you will find a list of examples that takes you through the basics of connecting to a local node, retrieving data from the Node and chain and execute transactions on the chain. It uses the ApiPromise interface.
 
 ## Prerequisites
 
-For the following examples, you need a local node. It is usually convenient testing with:
-
-```
-substrate --dev
+For the following examples, you need to install and run a local Substrate chain node.
```

Comment:
As above, no getsubstrate, the Substrate wiki should take care of how to run. (We really don't want to re-wrok the world)